### PR TITLE
private sector 99 dots updates

### PR DESCRIPTION
**Pull request by proteusvacuum**
Bunch of updates to the 99DOTS payload for 1.1 and phase 2a of eNikshay

@NoahCarnahan @mkangia 🐡 / 🐡 

Hold off on merge because I think it will make a bunch of conflicts in https://github.com/dimagi/commcare-hq/pull/15963 and that one is slightly more pressing (also I want to verify with 99DOTS that they are OK with us changing the format)

### CODE REVIEW

**Review by NoahCarnahan — APPROVED** [review]
All looks good

File: custom/enikshay/integrations/ninetyninedots/repeater_generators.py [hunk 1]
```diff
@@ -53,10 +74,11 @@ class PatientPayload(jsonobject.JsonObject):
     state_code = jsonobject.StringProperty(required=False)
     district_code = jsonobject.StringProperty(required=False)
     tu_code = jsonobject.StringProperty(required=False)
-    phi_code = jsonobject.StringProperty(required=False)
+    phi_code = jsonobject.StringProperty(required=False, exclude_if_none=True)
+    he_code = jsonobject.StringProperty(required=False, exclude_if_none=True)
```
Comment (mkangia) [inline custom/enikshay/integrations/ninetyninedots/repeater_generators.py]: Just for context what is `he`?

Comment (proteusvacuum) [inline custom/enikshay/integrations/ninetyninedots/repeater_generators.py]: "health establishment". It is the private sector equivalent of a PHI (although is slightly different and lives in a different place in the location hierarchy)

**Review by NoahCarnahan — APPROVED** [review]

**Review by NoahCarnahan — APPROVED** [review]
Latest changes look good

**Review by mkangia — COMMENTED** [review]
@proteusvacuum i added few comments but forgot to submit review. Submitting this now. Feel free to make a change that adds value.

**Pull request merged**